### DISCUSSION TITLE
Can TwoSampleMR run on a server without network? and how?

### DISCUSSION THREAD

**Discussion by lanxianmei** [MRCIEU/TwoSampleMR#257] [Q&A]

**Comment by explodecomputer** [comment]
Yes, you can read in local data to perform the analysis e.g. see https://mrcieu.github.io/TwoSampleMR/articles/exposure.html#reading-in-from-a-file-1 and https://mrcieu.github.io/TwoSampleMR/articles/outcome.html#using-local-gwas-summary-data-1
